eng/common/scripts/Test-Spelling.ps1

```diff
@@ -0,0 +1,87 @@
+[CmdletBinding()]
+Param (
+    [Parameter()]
+    [ValidateNotNullOrEmpty()]
+    [string] $TargetRef = 'master', 
```

Comment:
I'd prefer we default to something like https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/scripts/get-markdown-files-from-changed-files.ps1#L5 instead of master, especially given our upcoming plans to change the default branch to main.

Reply:
GitHub doesn't let me use more thumbs up here... 👍 👍 👍

Reply:
Because this script is written to be run on a dev box as well I wanted defaults that would work today. In this case I think removing the default is the best course of action so you don't end up with a default value of `origin/` when running on a dev box. 

The yaml template invoking the script has flexibility to put the context-appropriate variables in place.